freemarker-core/src/main/java/freemarker/template/Version.java

```diff
@@ -42,7 +42,6 @@ public final class Version implements Serializable {
     private final String originalStringValue;
     
     private final Boolean gaeCompliant;
-    private final Date buildDate;
```

Comment:
Admittedly this has near 0 practical importance, but... as `Version` has public constructor, and we allow passing in the build date, it would be more round if `getBuildDate()` still gives back the same value. Even if it will be almost always `null`.

Reply:
Ok than I misinterpreted your previous comment.

> I say, we just keep it in the `Version` API, but mark it as deprecated, and always return `null`. (It was documented that it can return `null`, and it's quite unlikely that anyone even calls that method, and also fails on `null`.)

Could you please clarify?

Do you mean we keep Version.java as it was before, but just remove the part in Configuration.java 

https://github.com/apache/freemarker/pull/103/commits/9a138b67dcc3d2de57d195d2b2e4d00b549a6239#diff-114f8ea56965362193c291c9a631f7c7860c71c8ec76a94b16f5f801267bd578

Reply:
Yes, `Version` need not change, we just that won't have the build date in `version.properties`.